testsuite/classlibrary/TestPatternProxy.sc

```diff
@@ -366,4 +366,19 @@ TestPatternProxy : UnitTest {
 
 
 	}
+
+	test_pbindef_arg_order {
+		var pairs;
+
+		Pbindef(\test).clear;
+		Pbindef(\test, \a, 1, \b, 2, \c, 3);
+
+		pairs = Pbindef(\test).source.pairs;
+		this.assertEquals(pairs[0,2..], [\a, \b, \c], "Pbindef key order should correspond to code");
```

Comment:
ok, so 1. and 2. are your two test methods :) each test should only test one thing. just for my own sanity, this PR was only designed to fix (2.) right?